File: Examples/Features/Formatting/Fill/SolidFill.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.Fill
{
    public class SolidFill : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];
            worksheet.Range["A2"].Interior.Color = Color.Green;
        }
    }
}

File: Examples.VB/Features/Formatting/Fill/SolidFill.vb
Imports System.Drawing

Namespace Features.Formatting.Fill
    Public Class SolidFill
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            worksheet.Range!A2.Interior.Color = Color.Green
        End Sub
    End Class
End Namespace
